modules/k3s/src/main/java/org/testcontainers/k3s/K3sContainer.java

```diff
@@ -46,8 +47,12 @@ public K3sContainer(DockerImageName dockerImageName) {
         tmpFsMapping.put("/var/run", "");
         setTmpFsMapping(tmpFsMapping);
 
-        setCommand("server", "--no-deploy=traefik", "--tls-san=" + this.getHost());
-        setWaitStrategy(new LogMessageWaitStrategy().withRegEx(".*Node controller sync successful.*"));
+        setCommand("server", "--disable=traefik", "--tls-san=" + this.getHost());
+        setWaitStrategy(
+            new LogMessageWaitStrategy()
+                .withRegEx(".*Node controller sync successful.*")
+                .withStartupTimeout(Duration.ofMinutes(5))
```

Comment:
so, 5 min is required? The max should be 1 min for good experience.

Reply:
Sorry, this were my own tests, I should have done these experiments on a separate branch, not on this PR.

I'm trying to figure out why local `./gradlew :k3s:test` works just fine locally, but not on GHA